doc/connection-to-the-internet-of-things/deep-dive.md

```diff
@@ -1,13 +1,11 @@
-Should you wish to learn much more about this, please check: 
+Should you wish to learn much more about this. Currently exists the following IoT Agents:
 
```

Comment:
Should you wish to learn much more about this, please check the following IoT Agents: